shared/tracker/action.render.js.flow

```diff
@@ -6,11 +6,13 @@ export type ActionProps = {
   loggedIn: boolean,
   state: SimpleProofState,
   currentlyFollowing: boolean,
-  username: ?string,
+  username: string,
```

Comment:
I'll look at the code in the morning but why do we need this last action
concept. Can't we just update the state to followed and it'll tender how we
want ?

On Tuesday, February 23, 2016, Chris Ball <notifications@github.com> wrote:

> In shared/tracker/action.render.js.flow
> <https://github.com/keybase/client/pull/2114#discussion_r53892260>:
>
> > @@ -6,11 +6,13 @@ export type ActionProps = {
> >    loggedIn: boolean,
> >    state: SimpleProofState,
> >    currentlyFollowing: boolean,
> > -  username: ?string,
> > +  username: string,
>
> Not sure why username allowed null, seems like we should always know it.
> Anyone know?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2114/files#r53892260>.
>